DnD-Montreal/session-tome
# Add new event accessors

# Description
Closes #435 

Adds accessors to the `Event` model that expose some statistics about the event. Statistics such as: Open seats, taken seats, seats left, and date range of the event.

## How to test
1. Via tinker Create an event (or find one in the seeded data)
2. Attach some sessions to that event (or ensure the seeded version has some)
3. Register some users / characters to some sessions on that event
4. Check the values of `$event->total_seats`, `$event->seats_taken`, `$event->seats_left`
5. Ensure that the above values based off the amount of seats available (sum of seats on sessions, count of registered users, etc)
6. Check the value of  `$event->scheduled_dates` it should be an array with 2 elements the first is the start, the second the end (times of first and last session)
